2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -819,7 +833,112 @@ Le task sono organizzate in tre colonne, rappresentanti lo stato:
+
+Gli intenti o i desideri di chi si oppone al gruppo o ad uno o più dei processi di ciclo di vita del sistema software (descritti in TODO) sono affrontati tramite il processo di gestione dei rischi TODO.
+
+Il negoziato tra le parti potrebbe essere richiesto per mediare posizioni mutualmente incompatibili, o a causa di vincoli o budget insufficiente. Anche la data di consegna prevista incide sulla realizzazione dei requisiti. Sarà sempre necessario consultare gli stakeholder coinvolti per raggiungere un accordo. Le decisioni saranno tracciate e rese disponibili agli stakeholder.
+
+==== Identificazione dei bisogni <identificazione-bisogni>
+Include l'elicitazione dei bisogni direttamente dagli stakeholder o dalla documentazione fornita da essi, oppure la raccolta di bisogni impliciti basati sul dominio applicativo ed i contesti tecnologico, legislativo, normativo.
+
+I bisogni degli stakeholder scaturiscono da fonti diverse. Il gruppo si impegna ad esplorare e valutare, al fine di identificare possibili bisogni, almeno questi frangenti:
+- Obiettivi di alto livello che il sistema dovrebbe conseguire;
+- Contributi concreti che il sistema dovrebbe apportare a beneficio degli stakeholder;
+- Scenari operativi, utili per limitare l'ambito e comprendere le aspettative e i bisogni;
+- Scenari operativi quotidiani, utili per assegnare una priorità ai bisogni;
+- Tipologie e caratteristiche degli utenti;
+- Ambiente operativo e contesto d'utilizzo;
+- Aspettative sulle prestazioni e la disponibilità del sistema;
+- Pratiche di business;
+- Norme, leggi o altri vincoli esterni.
+
+==== Definizione delle priorità
+// non mi piace
+Le preferenze espresse dagli stakeholder, coadiuvate dal processo di gestione delle decisioni (TODO), guidano la selezione e la prioritizzazione dei requisiti.
+
+==== Casi d'uso
+Si definisce un insieme di casi d'uso (anche detti _use case_, abbreviato in _UC_) che identifichi tutte le funzionalità attese.
+I casi d'uso sono definiti ed utilizzati nel documento di _Analisi dei Requisiti_. Sono fonte di bisogni e, indirettamente, di requisiti; sono un ausilio per l'esplorazione degli aspetti descritti nella @identificazione-bisogni.
+
+Ogni caso d'uso comprende:
++ Codice identificatore;
++ Titolo;
++ Descrizione;
++ Attore;
++ Precondizioni (opzionale);
++ Postcondizioni (opzionale);
++ Scenario principale;
++ Scenari alternativi (opzionale);
++ Generalizzazioni (opzionale).
+
+Il codice identificatore assume l'aspetto `UC-X.Y`, dove `UC-` è la radice del codice; `X` è una cifra positiva crescente di cifre che identifica un caso d'uso; `Y` è una cifra positiva crescente, attiva solo per le generalizzazioni di uno stesso caso d'uso.
+
+I casi d'uso sono arricchiti con diagrammi realizzati secondo la sintassi Unified Modeling Language (UML) 2.0.
```

Comment:
ho notato che in alcuni casi segni alcune parole in italic (come _use case_), però ho controllato e di questa convenzione stilistica non si trova traccia nelle norme. Quindi 1) bisogna normare questo comportamento 2) se viene usato per riferirsi a parole in lingua inglese perchè non segnare nello stesso modo Unified Modeling Language (UML) 2.0? 
PS: mi è venuto in mente che probabilmente era per segnare le parole del glossario, in questo caso allora vale la cosa inversa, togliere tutte le parole in italic

Reply:
Hai ragione, non è normato. Non è neanche detto che sia una buona idea adottare questa convenzione e infatti ho tolto il corsivo da UC e company.
L'ho tenuto (e vorrei proporlo nelle norme) però per indicare i titoli dei paragrafi e i documenti. In ogni caso ci saranno sicuramente formattazioni out of spec nel resto del documento che sistemeremo prima della consegna

Reply:
https://github.com/typst/typst/issues/2695
forse questa potrebbe essere una soluzione adatta al problema